reporter/otlp_reporter.go

```diff
@@ -146,16 +146,13 @@ func (r *OTLPReporter) SupportsReportTraceEvent() bool { return true }
 
 // ReportTraceEvent enqueues reported trace events for the OTLP reporter.
 func (r *OTLPReporter) ReportTraceEvent(trace *libpf.Trace,
-	timestamp libpf.UnixTime64, comm, podName,
-	containerName, apmServiceName string) {
+	timestamp libpf.UnixTime64, comm, apmServiceName string, _ util.PID) {
 	traceEvents := r.traceEvents.WLock()
 	defer r.traceEvents.WUnlock(&traceEvents)
 
 	key := traceAndMetaKey{
 		hash:           trace.Hash,
```

Comment:
Future PR: calculate the Hash on the backend side?

Reply:
Yeah: there's really no good reason to do this in the client, and the trace hashes are very high cardinality.